subprojects/pbcopper.wrap

```diff
@@ -1,5 +1,4 @@
 [wrap-git]
```

Comment:
Actually, you can remove the `pbcopper` dependency. The only reason it's here is because I thought it was needed by Pbbam. I just tested locally and it turns out that it's not. There's only a couple of lines in the `meson.build` and this wrap file to remove.